Exiled.Events/Commands/Reload/Permissions.cs

```diff
@@ -0,0 +1,51 @@
+﻿// -----------------------------------------------------------------------
+// <copyright file="Permissions.cs" company="Exiled Team">
+// Copyright (c) Exiled Team. All rights reserved.
+// Licensed under the CC BY-SA 3.0 license.
+// </copyright>
+// -----------------------------------------------------------------------
+
+namespace Exiled.Events.Commands.Reload
+{
+    using System;
+
+    using CommandSystem;
+    using Exiled.Events.Handlers;
+    using Exiled.Permissions.Extensions;
+
+    /// <summary>
+    /// The reload configs command.
+    /// </summary>
+    public class Permissions : ICommand
+    {
+        /// <summary>
+        /// Gets static instance of the <see cref="Permissions"/> command.
+        /// </summary>
+        public static Permissions Instance { get; } = new();
+
+        /// <inheritdoc/>
+        public string Command { get; } = "permissions";
+
+        /// <inheritdoc/>
+        public string[] Aliases { get; } = new[] { "perms" };
+
+        /// <inheritdoc/>
+        public string Description { get; } = "Reload permissions.";
+
+        /// <inheritdoc/>
+        public bool Execute(ArraySegment<string> arguments, ICommandSender sender, out string response)
+        {
+            if (!sender.CheckPermission("ee.reloadpermissions"))
+            {
+                response = "You can't reload permissions, you don't have \"ee.reloadpermissions\" permission.";
+                return false;
+            }
+
+            Exiled.Permissions.Extensions.Permissions.Reload();
+            Server.OnReloadedPermissions();
```

Comment:
You should move this call at the end of the `Permission::Reload` method.

Reply:
Check the summaries of OnReloaded... events

Reply:
Other events of reloaded something calls in command too

Reply:
I thought they were generic events, I don't know what could be the use case of those events, they're fine but I would have liked them more if made generic.